packages/react/.eslintrc.json

```diff
@@ -45,6 +45,7 @@
     "import/no-cycle": 0,
     "import/no-extraneous-dependencies": "off",
     "react/no-unknown-property": ["error", { "ignore": ["css"] }],
-    "react/prop-types": "off"
+    "react/prop-types": "off",
+    "react/no-unused-prop-types": "off"
```

Comment:
why was this required

Reply:
Since I am no longer directly using the props received in EmbeddedChat, but rather using them to initialize the config state, and then extracting the info from config state, the propTypes will throw a warning that props are not used. However, they are still being shown in propTypes.

Reply:
got it! thanks